test/hostname-policy.json

```diff
@@ -1,4 +1,8 @@
 {
+  "ExactBlacklist": [
```

Comment:
Filed an issue to add to integration test: https://github.com/letsencrypt/boulder/issues/2382